src/modules/sensors/sensors.cpp

```diff
@@ -2190,6 +2180,12 @@ Sensors::task_main()
 			orb_publish(ORB_ID(sensor_combined), _sensor_pub, &raw);
 		}
 
+		/* check parameters for updates */
+		parameter_update_poll();
```

Comment:
I moved these so they use the time between sensor updates and don't delay a sensor update.